packages/host/app/components/operator-mode/detail-panel-selector.gts

```diff
@@ -191,10 +219,27 @@ export default class Selector extends Component<Signature> {
         }
 
         .selector-item {
+          --icon-color: var(--boxel-highlight);
+
           display: flex;
           align-items: center;
+          flex-wrap: wrap;
+          text-wrap: nowrap;
```

Comment:
can you follow the css as in recent files so user knows when text exist. 

```
        overflow-wrap: anywhere;
        overflow: hidden;
```

I think this has been the overflow pattern for LHS. There was a recommendation to use ellipsis as the text-overflow but I think we need to set some max-width for the container. I think for now using the css above shud be fine so at least its consistent in the top (detail-panel) and bottom (recent-files)